scripts/install-flux.sh

```diff
@@ -3,10 +3,21 @@ set -ex
 
 ENV=$1
 CLUSTER_NAME=$2
-GIT_REPO=${3:-https://weaveworks.github.io/flux}
-VALUES=${4-deployments/weave-flux/values.yaml}
+VAULT_NAME=$3
+ENABLE_HELM_TLS=${4:false}
```

Comment:
why do we need this?

Reply:
To override in pipeline to enable for only environments where we want this

Reply:
sandbox and preview aren't supposed to have it enabled

I wouldn't default it though, I think it should always be set.
I don't like these optional arguments, we've already been bitten once by it

Reply:
👍

Reply:
Sure, i will get rid of the default in both scripts.